server/src/main/proto/spine/server/aggregate/aggregate.proto

```diff
@@ -70,8 +73,8 @@ message AggregateEventRecord {
     }
 }
 
-// The message for restoring an aggregate state.
-message AggregateStateRecord {
+// The message for restoring an aggregate state from a storage.
+message AggregateHistory {
```

Comment:
I'd expect a more abstract description of `AggregateHistory`. The current doc seems overly technical and restricts the scope in which the type may be used.

Reply:
As a matter of fact, we use this type not only for _read_, but also for _write_ operations.